desktop/src/com/unciv/app/desktop/UncivServer.kt

```diff
@@ -12,12 +12,31 @@ import java.io.File
 
 
 internal object UncivServer {
+    private var serverPort = 8080
+
     @JvmStatic
-    fun main(arg: Array<String>) {
+    fun main(args: Array<String>) {
+        args.forEach { arg ->
+            when {
+                arg.startsWith("-port=") -> {
+                    try {
+                        with(arg.removePrefix("-port=").toInt()) {
+                            if (this !in 1.rangeTo(65535)) println("'port' must in 1-65535")
```

Comment:
Spelling mistake :)
Also after refactor there should be only one error message for incompatible values, so that should be a problem